lib/tasks/install-blueprint.js

```diff
@@ -87,10 +89,16 @@ class InstallBlueprintTask extends Task {
   }
 
   _tryNpmBlueprint(name) {
+    let [, nameWithoutVersion, version] = name.match(/^(@?[^@]+)(?:@(.*))?$/);
```

Comment:
Hmm, why do we need to re-parse this manually? I'd much prefer to avoid being in the business of parsing npm's own arguments ourselves...

Does this work?

```js
let parsed = npa(name);
let nameWithoutVersion = parsed.name;
let version = parsed.fetchSpec;
```

Reply:
That worked perfectly. Great catch!